docs/Installation.rst

```diff
@@ -35,8 +31,14 @@ pip
        $ dnf install -y libvirt libvirt-devel libguestfs-tools \
            libguestfs-devel gcc libffi-devel openssl-devel qemu-kvm
 
-   C. Debian / Ubuntu - *TO-DO*
-   D. ArchLinux - *TO-DO*
+   C. Ubuntu 16.04+
```

Comment:
Actually should have been "Debian and its derivatives" but as I only tested this on Ubuntu 16.04, I changed it to Ubuntu 16.04 now.
When we test on Debian 9 - we can change it.